lib/measured/units/length.rb

```diff
@@ -1,8 +1,8 @@
 Measured::Length = Measured.build do
   unit :m, aliases: [:meter, :metre, :meters, :metres]
-  unit :cm, value: "0.01   m", aliases: [:centimeter, :centimetre, :centimeters, :centimetres]
-  unit :mm, value: "0.001  m", aliases: [:millimeter, :millimetre, :millimeters, :millimetres]
+  unit :cm, value: "1/100 m", aliases: [:centimeter, :centimetre, :centimeters, :centimetres]
+  unit :mm, value: "1/1000 m", aliases: [:millimeter, :millimetre, :millimeters, :millimetres]
   unit :in, value: "0.0254 m", aliases: [:inch, :inches]
-  unit :ft, value: "0.3048 m", aliases: [:foot, :feet]
-  unit :yd, value: "0.9144 m", aliases: [:yard, :yards]
+  unit :ft, value: "12 in", aliases: [:foot, :feet]
+  unit :yd, value: "3 ft", aliases: [:yard, :yards]
```

Comment:
I figure we only need one imperial unit with a mapping to a metric unit, the rest can be between imperial units.